tests/dummy/app/templates/components/scrolling-table.hbs

```diff
@@ -2,16 +2,16 @@
 <LightTable @table={{this.table}} @height="65vh" as |t|>
 
   {{!--
-    In order for `fa-sort-up` and `fa-sort-down` icons to work,
+    In order for `fa-sort-asc` and `fa-sort-desc` icons to work,
     you need to have ember-font-awesome installed or manually include
     the font-awesome assets, e.g. via a CDN.
   --}}
 
   <t.head
     @onColumnClick={{this.onColumnClick}}
     @iconSortable="fa fa-sort"
-    @iconAscending="fa fa-sort-up"
-    @iconDescending="fa fa-sort-down"
+    @iconAscending="fa fa-sort-asc"
```

Comment:
These are wrong, they should be `up` and `down`. I changed these already to `up` and `down`.

Reply:
fixed